Docs/telemetry.md

```diff
@@ -227,6 +227,12 @@ The event ping contains a list of events ([see event format on readthedocs.io](h
 |---------------------------------------------------------------------|----------|-----------|---------|------------------|---------------------------------------------------|
 | Allow user to search or add url from tab tray search button  | action   | tap | start-search-button | -n/a-          | -n/a-                     |
 
+### Url Bar  
+
+| Event                                                               | category | method    | object  | value            | extras                                            |
```

Comment:
If the code generation for the Glean SDK went correctly, it would have updated the `/docs/metrics.md` file and you should no longer have to manually document your telemetry like this.  This should only be necessary for legacy telemetry.